docs/source/conf.py

```diff
@@ -288,16 +288,34 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
+
 html_theme_options = {
-    "show_toc_level": 2,
     "use_edit_page_button": True,
+    "logo": {
+      "image_light": "_static/arrow.png",
+      "image_dark": "_static/arrow-dark.png",
+    },
+    "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
```

Comment:
How does this differ from the default?

Reply:
The default is `"navbar_end": ["navbar-icon-links"]` (https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/layout.html#header-sections).